packages/tools/devtools/devtools-core/src/test/types/validateDevtoolsCorePrevious.generated.ts

```diff
@@ -20,960 +20,897 @@ declare type MakeUnusedImportErrorsGoAway<T> = TypeOnly<T> | MinimalType<T> | Fu
+ * "Function_createDevtoolsLogger": {"backCompat": false}
  */
-declare type old_as_current_for_VariableDeclaration_devtoolsMessageSource = requireAssignableTo<TypeOnly<typeof old.devtoolsMessageSource>, TypeOnly<typeof current.devtoolsMessageSource>>
+declare type current_as_old_for_Function_createDevtoolsLogger = requireAssignableTo<TypeOnly<typeof current.createDevtoolsLogger>, TypeOnly<typeof old.createDevtoolsLogger>>
 
 /*
  * Validate backward compatibility by using the current type in place of the old type.
  * If this test starts failing, it indicates a change that is not backward compatible.
  * To acknowledge the breaking change, add the following to package.json under
  * typeValidation.broken:
- * "VariableDeclaration_devtoolsMessageSource": {"backCompat": false}
+ * "Variable_devtoolsMessageSource": {"backCompat": false}
  */
-declare type current_as_old_for_VariableDeclaration_devtoolsMessageSource = requireAssignableTo<TypeOnly<typeof current.devtoolsMessageSource>, TypeOnly<typeof old.devtoolsMessageSource>>
-
-/*
- * Validate forward compatibility by using the old type in place of the current type.
- * If this test starts failing, it indicates a change that is not forward compatible.
- * To acknowledge the breaking change, add the following to package.json under
- * typeValidation.broken:
- * "FunctionDeclaration_handleIncomingMessage": {"forwardCompat": false}
- */
-declare type old_as_current_for_FunctionDeclaration_handleIncomingMessage = requireAssignableTo<TypeOnly<typeof old.handleIncomingMessage>, TypeOnly<typeof current.handleIncomingMessage>>
+declare type current_as_old_for_Variable_devtoolsMessageSource = requireAssignableTo<TypeOnly<typeof current.devtoolsMessageSource>, TypeOnly<typeof old.devtoolsMessageSource>>
 
 /*
  * Validate backward compatibility by using the current type in place of the old type.
  * If this test starts failing, it indicates a change that is not backward compatible.
  * To acknowledge the breaking change, add the following to package.json under
  * typeValidation.broken:
- * "FunctionDeclaration_handleIncomingMessage": {"backCompat": false}
- */
-declare type current_as_old_for_FunctionDeclaration_handleIncomingMessage = requireAssignableTo<TypeOnly<typeof current.handleIncomingMessage>, TypeOnly<typeof old.handleIncomingMessage>>
-
-/*
- * Validate forward compatibility by using the old type in place of the current type.
- * If this test starts failing, it indicates a change that is not forward compatible.
- * To acknowledge the breaking change, add the following to package.json under
- * typeValidation.broken:
- * "FunctionDeclaration_handleIncomingWindowMessage": {"forwardCompat": false}
+ * "Function_handleIncomingMessage": {"backCompat": false}
  */
-declare type old_as_current_for_FunctionDeclaration_handleIncomingWindowMessage = requireAssignableTo<TypeOnly<typeof old.handleIncomingWindowMessage>, TypeOnly<typeof current.handleIncomingWindowMessage>>
```

Comment:
Is it expected that we have net fewer checks here than we did previously? I'm guessing we made changes to remove some redundancy in the checks, just want to make sure that's what's going on here.

Reply:
Yes. I removed what I consider some bad check, and added (fewer) of what I found to be missing checks. details in https://github.com/microsoft/FluidFramework/pull/21876